Assets/MixedRealityToolkit.SDK/Profiles/DefaultMixedRealityToolkitConfigurationProfile.asset

```diff
@@ -47,3 +47,4 @@ MonoBehaviour:
       Microsoft.MixedReality.Toolkit.Services.DiagnosticsSystem
   registeredServiceProvidersProfile: {fileID: 11400000, guid: efbaf6ea540c69f4fb75415a5d145a53,
     type: 2}
+  useServiceInspectors: 1
```

Comment:
>useServiceInspectors: 1 [](start = 2, length = 23)

Intentional change here? Feels like if we do this we should do it in its own change (more easily solo-revertable, not piggybacking on another change etc)